test/test_update_participant_family.py

```diff
@@ -0,0 +1,70 @@
+from test.testbase import DbIsolatedTest, run_as_sync
+from db.python.layers.participant import ParticipantLayer
+from db.python.layers.family import FamilyLayer
+
+
+class TestParticipantFamily(DbIsolatedTest):
+    """Test moving a participant from one family to another"""
+
+    @run_as_sync
+    async def setUp(self) -> None:
+        super().setUp()
+
+        fl = FamilyLayer(self.connection)
+        await fl.create_family(external_id='FAM01')
+        await fl.create_family(external_id='FAM02')
+
+        pl = ParticipantLayer(self.connection)
+        pid = await pl.create_participant(external_id='EX01', reported_sex=2)
+        await pl.create_participant(external_id='EX01_pat', reported_sex=1)
+        await pl.create_participant(external_id='EX01_mat', reported_sex=2)
+
+        await pl.add_participant_to_family(
+            family_id=1, participant_id=pid, paternal_id=2, maternal_id=3, affected=2
+        )
+
+    @run_as_sync
+    async def test_get_and_update_family_participant_data(self):
+        """
+        Tests getting and updating a participants family data
+        """
+        pl = ParticipantLayer(self.connection)
+
+        p = await pl.get_participants(project=1)
+        pid = p[0].id
+
+        fp_row = await pl.get_family_participant_data(family_id=1, participant_id=pid)
```

Comment:
This value can change between tests. You can make it more secure by storing the family ID returned by create_family (or making sure it returns the ID of he created family), and using that to query against.

Reply:
I've updated the participant and family IDs generated in setup to be attributes of the test instance so they can be referenced in the test functions, that should keep them consistent